docker/project-fortis-services/run-server.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
```

Comment:
Any chance you can add a `set -e` so that it exists on first failure?

```
#!/usr/bin/env sh
set -e
```

Reply:
Great comment. I'm usually the first one to advocate for Bash strict mode (`set -euo pipefail`). However, some of the commands below are expected to fail (e.g. when waiting for Cassandra to come up) so I think it's cleaner to not enable strict-mode here since otherwise we'd have to sprinkle a bunch of `|| true` across the code which may confuse folks with less Bash experience.